mkdocs/docs/HPC/getting_started.md

```diff
@@ -0,0 +1,259 @@
+
+Upload both files (`run.sh` and `tensorflow-mnist.py`) to your **Home Directory** and go back to your shell.
+
+!!! Info
+
+    As an alternative, you can use WinSCP (see [our section](connecting.md#winscp))
+
+{%- else %}
+
+On your local machine you can run:
+<pre><code>curl -OL https://raw.githubusercontent.com/hpcugent/vsc_user_docs/main/examples/tensorflow_mnist/tensorflow_mnist.py
+curl -OL https://raw.githubusercontent.com/hpcugent/vsc_user_docs/main/examples/tensorflow_mnist/run.sh
+</code></pre>
+
+Using the `scp` command, the files can be copied from your local host to your *home directory* (`~`) on the remote host (HPC).
+<pre><code>scp tensorflow_mnist.py run.sh {{userid}}{{ loginnode }}:~ </code></pre>
+<pre><code>ssh  {{userid}}@{{ loginnode }} </code></pre>
+
+!!! Info
+
+    For more information about transfering files or `scp`, see [tranfer files from/to hpc](connecting.md#transfer-files-tofrom-the-hpc).
+
+{%- endif %}
+
+When running `ls` in your session on the {{hpcinfra}}, you should see the two files listed:
+
+```shell
+$ ls
+run.sh tensorflow_mnist.py
+```
+
+!!! Warning
+
+    When you do not see these files, make sure you uploaded the files to your **Home Directory**.
+
+### Submitting a job
+
+Jobs are submitted and executed using job scripts. In our case **run.sh** can be used as a (very minimal) job script.
+
+A job script is a shell script, a text file that specifies the resources, 
+the software that is used (via `module load` statements), 
+and the steps that should be executed to run the calculation.
+
+Our job script looks like this:
+
+<center>-- run.sh --</center>
+
+```bash
+#!/bin/bash
+
+module load TensorFlow/2.11.0-foss-2022a
+
+time python tensorflow_mnist.py
+
+```
+<sub>As you can see this job script will run the Python script: **tensorflow_mnist.py**</sub>
+
+
+The jobs you submit are per default executed on **cluser/{{defaultcluster}}**, you can swap to another cluster by issuing the following command.
+
+```shell
+module swap cluster/{{othercluster}}
+```
+
+!!! Tip
+    
+    When submitting jobs with limited amount of resources, it is recommended to use the [debug/interactive cluster](interactive_debug.md#interactive-and-debug-cluster): `donphan`. 
+
+{%- if site == 'Gent' %}
+
+    To get a list of all clusters and their hardware, see <https://www.ugent.be/hpc/en/infrastructure>.
+
+{%- endif %}
+
+This job script can now be submitted to the cluster's job system for execution, using the qsub (**q**ueue **sub**mit) command:
+
+```shell
+$ qsub run.sh
+{{jobid}}
+```
+
+This command returns a job identifier (*{{jobid}}*) on the HPC cluster. This is a unique identifier for the job which can be used to monitor and manage your job.
+
+!!! Warning
+ 
+    Please take note that the module commands exclusively modify environment variables. For instance, using `module swap cluster/{{othercluster}}` will instruct `qsub` to submit the job to the {{othercluster}} cluster, 
```

Comment:
```suggestion
    Note that the module commands only modify environment variables. For instance, running `module swap cluster/{{othercluster}}` will update your shell environment so that `qsub` submits a job to the {{othercluster}} cluster, 
```